client/state/selectors/has-wp-admin-editor-conflicting-plugin.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import isPluginActive from 'state/selectors/is-plugin-active';
+
+// List of known plugins that replaces the default WP Admin editor.
+const deniedPluginsList = [ 'elementor' ];
```

Comment:
If we kept the array of slugs to check out of the selector, and always have them passed in, then the selector is more capable of other uses than our own here.